src/src/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -57,6 +57,8 @@
     private boolean mIsMultiResourceRefreshToken;
 
     private String mTenantId;
+    
+    private String mFamilyClientId;
```

Comment:
yup, we also have unit test for this. Also adding new fields into serializable class is a compatible change.